systemtest/src/main/java/io/strimzi/systemtest/resources/operator/BundleResource.java

```diff
@@ -20,7 +20,7 @@
 public class BundleResource {
     private static final Logger LOGGER = LogManager.getLogger(BundleResource.class);
 
-    public static final String PATH_TO_CO_CONFIG = "../install/cluster-operator/050-Deployment-strimzi-cluster-operator.yaml";
+    public static final String PATH_TO_CO_CONFIG = "../install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml";
```

Comment:
@Frawless Will these renamings play well with upgrade tests which use different versions?

Reply:
Interesting question, in upgrade tests, we change this file only for upgrade to HEAD, for older versions we basically do `kubectl apply -f /tmp/upgrade-54556445/strimzi-0.x.y/install/cluster-operator` and for HEAD Standa did proper changes from my POV. Let's try to run upgrade tests and we will see.